pom.xml

```diff
@@ -83,21 +212,6 @@
       </dependency>
     </dependencies>
 
-    <packaging>jar</packaging>
```

Comment:
If we can't build the jar anymore, a lot of the readme will need changing

Reply:
This was relocated to line 9, not removed